ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main.yaml

```diff
@@ -42,15 +51,29 @@ tests:
     pre:
     - ref: rosa-terraform-e2e-setup
 - as: e2e-periodic
+  cluster: build05
   cron: '@daily'
   steps:
     cluster_profile: aws-qe
     env:
       OPENSHIFT_VERSION: 4.13.0
+      POLARION_PROPERTIES: |
+        polarion-testrun-id: "20230528-0959"
+        polarion-custom-subteam: "kni"
+        polarion-custom-casecomponent: "installer"
+        polarion-custom-pillar: "acceleration"
+        polarion-custom-version: "4_13"
+        polarion-custom-products: "ocp"
+        polarion-custom-plannedin: "OCP_4_13_Z"
+      TEST_FILTERS: ~MicroShiftOnly&;~HyperShiftMGMT&;~CPaasrunOnly&;~NonPreRelease&;~VMonly&;~ProdrunOnly&;~StagerunOnly&;~DisconnectedOnly&;~Disruptive&;~Serial&;ROSA&
+      TEST_SCENARIOS: OLM|Security_and_Compliance|Operator_SDK|Network_Edge|Workloads|Image_Registry|Container_Engine_Tools|MCO|ETCD|PSAP|SDN|Cluster_Infrastructure|API_Server|Authentication|Cluster_Observability|LOGGING
     post:
     - ref: rosa-terraform-e2e-teardown
     pre:
     - ref: rosa-terraform-e2e-setup
+    test:
+    - ref: rosa-terraform-e2e-tests-ginkgo
+    - ref: rosa-terraform-e2e-tests-upload-polarion
```

Comment:
this step belongs to the ``post`` steps, as it's not actually doing the testing of the product

Reply:
pure testing... what u mean post step?

Reply:
``rosa-terraform-e2e-tests-upload-polarion`` is a post step

Reply:
no. it comes together with the ginkgo test.  why put in post?  (actually it should be a parameter to ginkgo tests and the code to be there. - but for sharing purposes, it has it's own ref step)

Reply:
post steps are for actions that are not doing any kind of verification: tearing-down resources, publishing results, gathering logs, etc.
the aspect of doing it always or not is just a side-effect, because it's usually the desired behavior

Reply:
I agree for teardown, gather_logs (do always, including failures) -  but not for publish_results (upload polarion is in this category).  I guess PROW flow, is flexible with what u put in pre/tests chains.
Anyway, I give up (not so significant)